sky/services/editing/editing.mojom

```diff
@@ -48,3 +48,13 @@ interface Keyboard {
   Show();
   Hide();
 };
+
+struct ClipData {
```

Comment:
Maybe ClipboardData ?  ClipData makes me think we're going to visually limit the extent of something with a clip.